backend/src/main/java/ca/bc/gov/app/exception/ValidationException.java

```diff
@@ -10,6 +10,9 @@
 @Getter
 @ResponseStatus(value = HttpStatus.BAD_REQUEST)
 public class ValidationException extends ResponseStatusException {
+
+  private static final long serialVersionUID = 8685785380234657208L;

```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data

Reply:
Thanks. My IDE was complaining for that, so I added it :-) The error was: "The serializable class ValidationError does not declare a static final serialVersionUID field of type long"